test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -394,11 +394,11 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			`.edit-post-layout__inserter-panel .block-editor-block-types-list button.editor-block-list-item-${ prefix }${ blockClass }, ${ deprecatedInserterBlockItemSelector }`
 		);
 
-		const insertedBlockSelector = By.css(
-			`.block-editor-block-list__block.${
-				initsWithChildFocus ? 'has-child-selected' : 'is-selected'
-			}[aria-label*='${ ariaLabel }']`
-		);
+		const insertedBlockSelectorStr = `.block-editor-block-list__block.${
+			initsWithChildFocus ? 'has-child-selected' : 'is-selected'
+		}[aria-label*='${ ariaLabel }']`;
+
+		let insertedBlockSelector = By.css( insertedBlockSelectorStr );
```

Comment:
One of the things I wanted to refactor at some point - the `By` interface has "locator strategies" that return `locator`s, not `selector`s. A `selector` in this case is actually the string passed to the locator strategy (`css()` in this case). We use the selector naming for locators everywhere, so we should keep doing that to stay consistent, IMO. Just thought I'd mention the "locators" for future reference :) 

https://www.selenium.dev/documentation/en/getting_started_with_webdriver/locating_elements/